addons/website/views/website_templates.xml

```diff
@@ -408,8 +408,8 @@
 <template id="publish_management">
     <div groups="base.group_website_publisher" t-ignore="true" class="pull-right css_editable_mode_hidden" t-att-style="style or ''">
         <div t-attf-class="btn-group js_publish_management #{object.website_published and 'css_published' or 'css_unpublished'}" t-att-data-id="object.id" t-att-data-object="object._name" t-att-data-controller="publish_controller">
-            <button class="btn btn-danger js_publish_btn">Not Published</button>
-            <button class="btn btn-success js_publish_btn">Published</button>
+            <button class="btn btn-danger js_publish_btn" data-loading-text="Loading...">Not Published</button>
+            <button class="btn btn-success js_publish_btn" data-loading-text="Loading...">Published</button>
```

Comment:
Please don't use the `.button()` of bootstrap. The `data-loading-text`you add is not translatable. You should manually set the label in the js, using openerp `_t()` method for translation.